### PR TITLE
fix(localizations):Update Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by youyou-sudo**
Update Simplified Chinese according to the latest en-US version

> This is my first pr and if I've done anything wrong then I'm very sorry

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Review by tmilewski — APPROVED** [review]

**Comment by tmilewski** [comment]
@youyou-sudo Thank you for this PR! Would you mind adding a changeset to the PR? Here's more information on how to go about doing that: https://github.com/clerk/javascript/blob/main/docs/CONTRIBUTING.md#changesets

Thanks again!

File: packages/localizations/src/zh-CN.ts [hunk 1]
```diff
@@ -196,7 +196,7 @@ export const zhCN: LocalizationResource = {
         subtitle: '允许用户根据已验证的电子邮件域名自动加入组织或请求加入。',
         title: '已验证域名',
       },
-      successMessage: '组织已更新。',
+      successMessage: '组织信息已更新。',
```
Comment (tmilewski) [inline packages/localizations/src/zh-CN.ts:199]: @youyou-sudo Let's keep this the same. According to Google Translate, the existing message specifically calls out organization which is more ideal.

Comment (youyou-sudo) [reply packages/localizations/src/zh-CN.ts:199]: Thank you for your feedback! As my first PR, I am glad to receive your guidance.
Also, I have added a Changeset. Could you please take a look? Thank you!

Comment (tmilewski) [reply packages/localizations/src/zh-CN.ts:199]: @youyou-sudo This is great, thank you!

**Comment by youyou-sudo** [comment]
> Thanks again, @youyou-sudo!

You're welcome! Happy to contribute!

**Pull request merged**